.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Have a description on what is this about? Also should we have wso2 licenses added here?

Reply:
Resolved by e2c7a32